app/client/components/user/CreateUser.tsx

```diff
@@ -107,23 +116,27 @@ class CreateUser extends React.Component<Props> {
         return <CreateUserForm
           users={this.props.users}
           transferAccountType={transferAccountType}
+          transferUsages={this.props.transferUsages}
           onSubmit={(form: Form) => this.onCreateUser(form)}
         />
       }
     }
   }
 }
 
+//TODO: why doesn't ReduxState work correctly
```

Comment:
might be because there's a mix of ts and js reducers..? but it means we can't get the full benefits of typescript with react/redux here, which is unfortunate

Reply:
what goes wrong exactly?